libgalois/include/katana/TypedPropertyGraph.h

```diff
@@ -270,6 +270,12 @@ class TypedPropertyGraphView : public PGView {
       const std::vector<std::string>& edge_properties);
   static Result<TypedPropertyGraphView<PGView, NodeProps, EdgeProps>> Make(
       PropertyGraph* pg);
+  static Result<TypedPropertyGraphView<PGView, NodeProps, EdgeProps>> Make(
+      PropertyGraph* pg, const PGView& pg_view,
+      const std::vector<std::string>& node_properties,
+      const std::vector<std::string>& edge_properties);
+  static Result<TypedPropertyGraphView<PGView, NodeProps, EdgeProps>> Make(
+      PropertyGraph* pg, const PGView& pg_view);
```

Comment:
Is passing the `pg` here required? It would make sense for the view to have all the information needed to wrap it as a type graph view.

Reply:
Agree with Arthur. Ditto about dependent arguments. Doesn't `PGView` have the `PropertyGraph`? What if the one it has is different from the one passed?

Reply:
yeah, will update this